pkg/apis/machine/v1beta1/machine_types.go

```diff
@@ -37,6 +37,10 @@ const (
 	// MachineClusterIDLabel is the label that a machine must have to identify the
 	// cluster to which it belongs.
 	MachineClusterIDLabel = "machine.openshift.io/cluster-api-cluster"
+
+	// PreserveInstanceAnnotation prevents a VM from being deleted by the
```

Comment:
is there a justification why we need this atm? If not, I'd prefer to not introduce it in this PR